.github/actions/build/action.yml

```diff
@@ -0,0 +1,58 @@
+name: "Build Webgreeter"
+description: "Builds webgreeter"
+inputs:
+ target-distro:
+     required: true
+     description: "Target distro"
+runs:
+  using: "composite"
+  steps:
+    -
+      name: Checkout
+      uses: actions/checkout@v2
+    -
+      name: Install dependencies
+      shell: bash
```

Comment:
Is `shell: bash` required? I guess the default one is enough.

Reply:
Looks like `shell` is required here for some reason. Without it i got `Error: /home/runner/work/web-greeter/web-greeter/./.github/actions/build/action.yml (Line: 14, Col: 7): Required property is missing: shell`. I also don't know much about actions so maybe i missed someting :o

Reply:
Well, it's not a problem, just curious~